python/graphstorm/dataloading/dataset.py

```diff
@@ -425,6 +464,10 @@ def get_blocks_edge_feats(self, input_blocks, efeat_fields, device='cpu'):
         .. versionadded:: 0.4.0
             Add ``get_blocks_edge_feat`` in 0.4.0 to support edge features in message passing.
 
+        .. versionchanged:: 0.5
+            Moved the implementatino of this function out of `GSgnnData` class in v0.5 to support
+            real-time node-level inference.
```

Comment:
So we moved the function in order to allow it to be called without a GSgnnData object?

Reply:
Yes, the `GSGnnNodePredictionRealtimeInferrer` does not use `GSgnnData`, but a DGLGraph. So need this function to be independent.